### PR TITLE
updated the README file

### PR DESCRIPTION
**Pull request by Nebyt**

### CODE REVIEW

File: apps/wallet-mobile/README.md [hunk 1]
```diff
@@ -1,664 +1,69 @@
...
-nvm install 16.5.0
-nvm alias default 16.5.0
-```
+### Preparation
 
-- Install yarn:
+- First of all it is necessary to install [`brew`](https://brew.sh/). It will help a lot further.
+- Install Xcode
+- Install the stable version of Xcode command line tools through the web site or run the command `xcode-select --install`
+- Install `nvm` to that run the command `brew install nvm`. The node version `v16.8.0` is reqired.
```
Comment (stackchain) [inline apps/wallet-mobile/README.md]:
```suggestion
- Install `nvm` to that run the command `brew install nvm`. The node version `v16.19.0` is required.
```

Comment (stackchain) [reply apps/wallet-mobile/README.md]: dunno about the min - version for `nvm`

File: apps/wallet-mobile/README.md
Comment (stackchain) [inline apps/wallet-mobile/README.md:1]: maybe breakdown the installation in `common steps`, and `plataform x, y` ?

Comment (Nebyt) [reply apps/wallet-mobile/README.md:1]: It is the old README. I don't want to remove it till the new instruction is ready.

Comment (Nebyt) [reply apps/wallet-mobile/README.md:1]: But it is already split...

Comment (Nebyt) [reply apps/wallet-mobile/README.md:1]: Or do you mean move it to a different files?

**Pull request closed without merging**